practices/pianissimo-pp/practica4/tokenizacion.py

```diff
@@ -0,0 +1,111 @@
+# ---
+# Ejercicio 1
+# ---
+
+from collections import Counter
+import re
+import math
+
+def write_plain_text_corpus(raw_text: str, file_name: str) -> None:
+    with open(f"{file_name}.txt", "w") as f:
+        f.write(raw_text)
+
+def plain_text_to_list(file_name:str) -> list:
+    data = []
+    with open(f"{file_name}.txt", "r") as file:
+        data = file.read().split(" ")
+    return(data)
+
+def calculate_entropy(corpus: list[str]) -> float:
+    words_counts = Counter(corpus)
+    total_words = len(corpus)
+    probabilities = {word: count / total_words for word, count in words_counts.items()}
+    entropy = -sum(p * math.log2(p) for p in probabilities.values())
+    return entropy
+
+# Brown
+
+import nltk
+from nltk.corpus import brown
+nltk.download('brown')
+
+brown_words = [word for word in brown.words() if re.match("\w", word)]
+write_plain_text_corpus(" ".join(brown_words), "./practices/pianissimo-pp/practica4/brown_plain")
```

Comment:
Nuevamente, el path es relativo. No es necesario definir toda la ruta ya que el script esta ya dentro de `practica4`.

En todo caso sugiero crear una variable global `PATH` con la que se pueda modificar rápidamente este comportamiento

```suggestion
write_plain_text_corpus(" ".join(brown_words), "brown_plain")
```